utils.py

```diff
@@ -0,0 +1,21 @@
+def get_progress_emoji(ratio):
```

Comment:
If there are 10 levels, that is good choice.
But, we have just 5 levels, so the list may have duplicated items up to 4. it seems not clean.